libs/blocks/global-navigation/global-navigation.js

```diff
@@ -523,24 +523,30 @@ class Gnav {
 
     const onAnalyticsEvent = (data) => {
       if (!data) return;
-
       const getInteraction = () => {
-        const {
-          event: { type, subtype } = {},
-          source: { name } = {},
-          content: { name: contentName } = {},
-        } = data;
-
-        switch (`${name}|${type}|${subtype}|${contentName || ''}`) {
-          case 'profile|click|sign-in|':
+        const { workflow, type, subtype } = data.event || data;
+        const contentName = data.content?.name;
+
+        switch (`${workflow}|${type}|${subtype}${contentName ? `|${contentName}` : ''}`) {
+          case 'UNC|click|icon':
+            return 'Open Notifications panel';
+          case 'UNC|click|link':
+            return 'Open Notification';
+          case 'UNC|click|markRead':
+            return 'Mark Notification as read';
+          case 'UNC|click|dismiss':
+            return 'Dismiss Notifications';
+          case 'UNC|click|markUnread':
+            return 'Mark Notification as unread';
```

Comment:
Spaces, dots, vertical bars | .. so hard to spot any pattern in how those analytics labels are created. Do you have any context on what the "normal" case for the attribute formatting is? (Just out of curiosity)

Reply:
[These requirements](https://wiki.corp.adobe.com/pages/viewpage.action?spaceKey=marketingtech&title=Universal+Nav+on+Adobe.com+-+Custom+Tracking) had to be followed regarding the analytics labels, and, unfortunately, they are inconsistent 😔